src/components/Filters/index.js

```diff
@@ -187,10 +188,19 @@ class Filters extends Component {
     const inputModels = sieve.makeFilterInputs(this.props.schema)
 
     if (attribute === 'name' && update.name !== value) {
+      const model = inputModels[value]
       update.name = value
-      update.operator = inputModels[value].availableOperators[0]
-      update.label = inputModels[value].label
-      update.value = ''
+      update.operator = model.availableOperators[0]
+      update.label = model.label
+      if (model.type === 'Date Time') {
+        update.value = moment().format('YYYY-MM-DDTHH:mm')
+      } else if (model.type === 'Date') {
+        update.value = moment().format('YYYY-MM-DD')
+      } else if (model.type === 'Time') {
+        update.value = moment().format('HH:mm')
```

Comment:
How about moving the format strings to properties with default values like you did for the respective pine types?

Reply:
Unfortunately, this doesn't work as the value sent to the component is defined in the `edit` object. If the `Display` component in the pine type has a default value, then the display will show a value, but when you hit save, the rule won't have that value in it